tuna/miopen/miopen_tables.py

```diff
@@ -569,6 +569,37 @@ class BNBenchmark(BASE, BenchmarkMixin):
   config = Column(Integer, ForeignKey("bn_config.id"), nullable=False)
 
 
+class SolverAnalyticsResults(BASE):
+  """Table to store results from running SolverAnalytics"""
+  __tablename__ = "solver_analytics_results"
+
+  # each row in table must correspond to a unique convolution problem
+  __table_args__ = (UniqueConstraint("golden_v",
+                                     "filter",
+                                     "padding",
+                                     "stride",
+                                     "layout",
+                                     "precision",
+                                     "direction",
+                                     name="uq_idx"),)
+  
+  # columns definitions
+  golden_v = Column(Integer, nullable=False)
+  filter = Column(String(32), nullable=False)
+  padding = Column(String(32), nullable=False)
+  stride = Column(String(32), nullable=False)
+  dilation = Column(String(32), nullable=False)
```

Comment:
im pretty sure these need to be ints. Please check the conv_config table in tuna to see how we define these.

Reply:
They are going to be strings like "32x64x64"

Reply:
oh ok, if thats the case for all of them they can stay strings